src/features/common/Layout/ProjectsLayout/ProjectsLayout.module.scss

```diff
@@ -31,3 +31,36 @@
   border-radius: 16px;
   overflow: hidden;
 }
+
+// styles for mobile layout
+
+.mobileProjectsLayout {
+  width: 100%;
+  height: 100vh;
+  position: relative;
+  padding: 100px 5px 20px;
+  display: flex;
+  justify-content: center;
+
+  &.compact {
+    padding: 0px;
+  }
+}
+
+.mapButton {
+  position: absolute;
+  top: 100px;
+  right: 20px;
+  z-index: 1;
+}
+
+.projectlistAndDetail {
+  background: grey;
+  height: 100%;
+  width: 94%;
```

Comment:
1. Just for consistency, let's call this `mobileContentContainer`.
2. Any reason to keep width: 94%? If we do that, the space on the sides (padding) becomes variable depending on the screensize. Avoid adding an arbitrary value like this, and if it is needed, add a comment to make the reason clear.